orcoursetrion/tests/test_cmd.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""
+Test command line parsing and actions
+"""
+import sys
+
+import mock
+
+from orcoursetrion.cmd import execute
+from orcoursetrion.tests.base import TestGithubBase
+
+
+class TestGithubCommand(TestGithubBase):
+    """Verify github commands via command line"""
+
+    def test_cmd_create_export_repo(self):
+        """
+        Command line test of create_export_repo
+        """
+
+        sys.argv = [
+            'orcoursetrion', 'create_export_repo',
+            '-c', self.TEST_COURSE,
+            '-t', self.TEST_TERM,
+            '-d', self.TEST_DESCRIPTION,
+        ]
```

Comment:
I didn't bother since every other test that needs it will override it too, but you are right, it would be cleaner that way.  I'll patch it.